polkadot/xcm/xcm-executor/src/lib.rs

```diff
@@ -1212,9 +1212,9 @@ impl<Config: config::Config> XcmExecutor<Config> {
 				);
 				Ok(())
 			},
-			HrmpNewChannelOpenRequest { .. } => Err(XcmError::Unimplemented),
-			HrmpChannelAccepted { .. } => Err(XcmError::Unimplemented),
-			HrmpChannelClosing { .. } => Err(XcmError::Unimplemented),
+			HrmpNewChannelOpenRequest { sender, max_message_size, max_capacity } => Config::HrmpHandler::handle_new_channel_open_request(sender, max_message_size, max_capacity),
```

Comment:
@franciscoaguirre Do we need to handle `Config::TransactionalProcessor::process` here?

Reply:
Only if we do anything with storage that might need to be rollbacked if it fails. Sounds unlikely. Is this something that has happened in Polimec?

Reply:
I think as the trait implementations are quite generic, I could see implementations changing storage. We are currently using it change some values in storage. So not using TransactionalProcessor would force the implementations to never error during or after storage changes, so I'm in favor of wrapping with TransactionalProcessor.